src/libratbag-data.c

```diff
@@ -515,7 +515,9 @@ ratbag_device_data_new_for_id(struct ratbag *ratbag, const struct input_id *id)
 			goto out;
 	}
 
-	if (!data)
+	if (id->vendor == 0x046d && (id->product >> 8) == 0xc5)
```

Comment:
we have `USB_VENDOR_ID_LOGITECH` for the vendor ID which makes a little bit more obvious. Otherwise I'd probably go with this:

```c
/* The 0xc5* range is reserved for receivers */
if (... id->product & 0xff00 == 0xc500)
```
The only reason here is that 0xc5 is less obviously a 2-byte match for 0xc52b and others than 0xc500. But this is definitely personal preference, so feel free to go with what you think is better.